BaseFilter/sections/specific.txt

```diff
@@ -20,6 +20,18 @@ triblive.com###storyContent > div[data-type="float"][data-stn-player]
 ! !SECTION: Temporary
 !##################
 !
+! Ad-Shield ad reinsertion
+!#if (!adguard_ext_safari && !adguard_app_ios && !adguard_ext_android_cb && !ext_ublock)
```

Comment:
```suggestion
!#if (!adguard_ext_safari && !adguard_app_ios && !ext_ublock)
```

`AdGuard Content Blocker` (not AG for Android) can apply only basic rules. `$$` or `#%#` and others automatically removed from filter version for AdGuard Content Blocker.

Btw you can use this button if suggestion is ok

![image](https://github.com/AdguardTeam/AdguardFilters/assets/8361299/df31e634-1597-4454-8973-af40d41121d6)

and/or press `Resolve conversation`.

Reply:
Hmm nothing happened after i pressed the button. Changed manually now.

Reply:
Hm. Weird.
Thanks.

Reply:
> `AdGuard Content Blocker` (not AG for Android) can apply only basic rules. `$$` or `#%#` and others automatically removed from filter version for AdGuard Content Blocker.

Might be safer to include unless we are sure we never add classic rules, as applying only classic rules without advanced ones can harm.